src/decoder.c

```diff
@@ -239,6 +239,26 @@ int nanocbor_get_tag(nanocbor_value_t *cvalue, uint32_t *tag)
     return res;
 }
 
+int nanocbor_get_decimal_frac(nanocbor_value_t *cvalue, int32_t *e, int32_t *m)
+{
+    int res = NANOCBOR_NOT_FOUND;
+    uint32_t tag;
+    nanocbor_get_tag(cvalue, &tag);
```

Comment:
The return value of this call should be checked. the value of `tag` is undefined if the decoded byte is not a tag type.